app/views/pages/show.html.erb

```diff
@@ -1,3 +1,5 @@
 <main>
-  <%= raw @page.content %> 
+  <div class="article">
```

Comment:
`row` has a `max-width: 75rem` which is too wide for us. So I see no benefits of adding it.